### DISCUSSION TITLE
Why is the monte carlo simulation results so weird?

### DISCUSSION THREAD

**Discussion by lucas-sys-cmd** [LCA-ActivityBrowser/activity-browser#1329] [General]
Recently, I'm doing a research on AB and I have created several activities to simulate processes in real world. In these activities, I used parameters and formulas to quantify the amout of technosphere flows and biosphere flows. Then I used the created activities to combine the studied system. The results of LCA are quite normal and close to the results of other software. However, the median value of Monte Carlo simulation results was more than 2000 times larger. 
I am wondering why this happens. Could it be because I used parameters in self-create activities for processes but not in the combined activity for the system. Therefore, when performing Monte Carlo, the parameter uncertainties are actually the technosphere and bioshere uncertainties and parameter uncertainties can not be included.

![image](https://github.com/user-attachments/assets/221cee45-bc6c-44bc-a40d-a5e9b4b7f5d8)
![image](https://github.com/user-attachments/assets/e624847d-0198-4c98-8613-be7a21a9297a)
![image](https://github.com/user-attachments/assets/72a8962f-e3e7-4bb2-92c3-e8bbf9ffdc87)
![image](https://github.com/user-attachments/assets/25f0a24d-e41e-4532-a3f7-96decdc7d5b2)
![image](https://github.com/user-attachments/assets/fcc7a77b-eef8-4bf4-981b-a2543d2db5f0)

**Comment by maw-cu** [comment]
Hi there,

I have been through this and probably if you go through the issues of activity browsers here on GitHub, you will find the developers acknowledging (directly/indirectly) that the montecarlo simulation framework is problematic.

I will give you a very straightforward suggestion, do not use it. Instead, use the scenario-based LCA feature and in the excel file to be imported, apply yourself the random sampling technique and put the values for your modelling parameters. You will always find online excel-based approaches to generate random values for your modelling parameters in function of the statistical distribution of the uncertainty.

I use the activity browser v2.6.9, because it is the latest version that is convenient for this approach. I don't know why, but the developers made scenario-based modelling from v2.7.0 onwards very difficult and perhaps requiring more than just the input value so I have stopped at v2.6.9. Maybe it is not true anymore for later versions, but I gave up trying.

If you need help with performing a global sensitivity analysis for the montecarlo simulation results, please contact me at
mohammad.abdelbaky@kuleuven.be
